EpionHealth/interview
# Jira num epion health

Assumptions
- The original specs are correct. The user flow is what we expect. 
- Professionals will style the frontend code 😅 
- Visiting root_path patient are already logged in (inputting their social or some identifier to find the user)
- The Patient id matches the dummyjson user id (In real life, theres an Patient external_id and that would match the id for the API user id
- From screener questionaire page the answers `Not at all`, `Several days`, etc are all the same for each screener. If this is not true, then we'ed want to store these questionaire option in the db maybe something like Screener -> Questions -> QuestionOptions and display the question options
- There's a different process to create Users, Screeners, and Questions (I did it through console)

How to test
1. rails db:seed
2. run rails s
3. localhost:3000
4. you should be able to visit the page and submit a form
